yt/visualization/plot_window.py

```diff
@@ -864,8 +864,10 @@ def _setup_plots(self):
             axis_index = self.data_source.axis
 
             xc, yc = self._setup_origin()
-
-            if self._axes_unit_names is None:
+            if self.ds.unit_system.name.startswith("us"):
```

Comment:
Why `"us"`?  Is that because it's short for "unit system"?

Reply:
yes. That's the workaround I found. From the example I experimented with, `self.ds.unit_system.name` seems to start either with `"cgs_us"`, `"mks_us"` or just `"us"` , depending on which value is used to set the unit system at load time `yt.load(..., unit_system=<"cgs"|"mks"|"code">)`. If there are more possible cases I should try them out before we merge.

Reply:
I don't think we need to do that!  I think this PR is probably good to go.